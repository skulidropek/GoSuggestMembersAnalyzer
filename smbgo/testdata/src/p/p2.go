package p

var saveRef int

func useIdentifier() {
	_ = saveRe1f // want "undefined identifier"
}
