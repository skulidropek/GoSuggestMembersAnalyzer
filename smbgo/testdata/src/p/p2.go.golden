package p

var saveRef int

func useIdentifier() {
	_ = saveRef // want "undefined identifier"
}
